exercises/proverb/src/main/java/Proverb.java

```diff
@@ -0,0 +1,7 @@
+class Proverb {
```

Comment:
This class needs to have a signature that matches what is expected in the tests, so it should have the signature for all the methods that are used in the tests and a constructor which matches the one used in the test (or no constructor if the one used in the tests takes no argument, since a constructor with no arguments is provided by default in Java) :)